README.md

```diff
@@ -1 +1,37 @@
-# node-patterns-concurrency-lab1
+# Async Lab
+
+## Objectives
+
+1. Implement your own async loop
+1. You cannot use async or some other library
+1. You need to make 10 requests to another service and wait until all of them responded before rendering
+1. Utilize Storify API to fetch images
```

Comment:
It looks like Storify isn't used anywhere in this lab -- should we cut this objective?